features/product/viewing_products/viewing_different_price_for_different_product_variants_selected_with_options.feature

```diff
@@ -21,10 +21,4 @@ Feature: Viewing different price for different product variants selected with op
         When I view product "Wyborowa Vodka"
         And I set its volume to "0,7L"
         Then I should see the product price "$25.00"
-
-    @ui @javascript
-    Scenario: Viewing an unavailable message if selected option combination is unavailable
```

Comment:
Because of https://github.com/Sylius/Sylius/pull/11241#discussion_r404822326.
Given that the "match" variant selection method now only show "avaliable" options is not possible anymore to select an unavailable combination.

Reply:
What will happen if we will have matrix option 2 x 2, and only diagonal variants will be available? 
E.g
| Size\Color  | Yellow  |  Blue |
|---|---|---|
| S  | X |   |
|  L |   | X |

What will be displayed to the customer in select boxes?

Reply:
Both values for both options because there's at least one enabled variant for at least one value for  Yellow, one for Blue, one for S and one for L. Do you think that a Behat scenario is needed to verify that?

Reply:
I would say so. Maybe this scenario should be improved, but this would be the exact case when someone would choose small blue from my previous example.

Reply:
Ok I'll do it!

Reply:
Done: https://github.com/Sylius/Sylius/pull/11241/commits/df192a303e3f674b113df8261cbb787633b8f637